frontend/src/pages/Facharbeit/FacharbeitApplicationForm.tsx

```diff
@@ -0,0 +1,159 @@
+        <h1>Facharbeit beantragen</h1>
+      </div>
+      <Paper style={{ display: 'flex', width: '90vw', padding: '2vw' }}>
+        <div
+          id="studentAndTopicInputs"
+          style={{
+            display: 'flex', flexDirection: 'column', width: '60%', marginRight: '3vw',
+          }}
+        >
+          <div style={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
+            <h2>Schüler:in</h2>
+            <div style={{ display: 'flex', justifyContent: 'space-between', width: '100%' }}>
+              <TextField label="Name" variant="outlined" style={{ width: '50%' }} onChange={handleStudentNameInputChange} />
+              <TextField label="Klasse" variant="outlined" style={{ width: '30%' }} onChange={handleStudentClassInputChange} />
+            </div>
+          </div>
+          <TextField
+            label="Thema"
+            variant="outlined"
+            style={{ width: '100%', marginTop: '3vh' }}
+            multiline
+            rows={10}
+            onChange={handleTopicInputChange}
+          />
+        </div>
+        <div id="teacherInputsAndButtons" style={{ display: 'flex', flexDirection: 'column', width: '40%' }}>
+          <h2> </h2>
+          <FormControl style={{ width: '100%', marginTop: '32px' }} variant="outlined">
+            <InputLabel>Fach</InputLabel>
+            <Select
+              label="Fach"
+              onChange={handleSubjectInputChange}
+            >
+              {subjectList.map((subject) => (
+                <MenuItem value={subject.id}>{subject.name}</MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl style={{ width: '100%', marginTop: '3vh' }} variant="outlined">
+            <InputLabel>Betreuende Lehrkraft</InputLabel>
+            <Select
+              label="Betreuende Lehrkraft"
+              onChange={handleBetreuendeLehrkraftInputChange}
+            >
+              {betreuendeLehrkraefteList.map((lehrkraft) => (
+                <MenuItem value={lehrkraft.id}>{lehrkraft.name}</MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl style={{ width: '100%', marginTop: '3vh' }} variant="outlined">
+            <InputLabel>Unterrichtende Lehrkraft</InputLabel>
+            <Select
+              label="Unterrichtende Lehrkraft"
+              onChange={handleUnterrichtendeLehrkraftInputChange}
+            >
+              {unterrichtendeLehrkraftList.map((lehrkraft) => (
+                <MenuItem value={lehrkraft.id}>{lehrkraft.name}</MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <Button style={{ marginTop: '3vh' }} variant="contained" color="primary" onClick={handleSubmitFacharbeit}>
+            Facharbeit einreichen / ändern
+          </Button>
+          <CreatePDFButton style={{ marginTop: '3vh' }} onClick={handleGeneratePDF} />
+        </div>
+      </Paper>
+      <Paper style={{
```

Comment:
den Teil mit den Dateien könnte man in eine extra Komponente verpacken

Reply:
Meinst du den handleGeneratePDF?

War ich kurz davor. hab ja schon ein createPDF button. da kann man dan props übergeben und ein template und der button kümmert sich ums generieren. wollte mich jetzt aber nicht festlegen, solange wir noch nicht wissen wie wir die PDFs nachher generieren

Reply:
ich meine nur den Teil unten wo die Datei angezeigt wird

Reply:
Ist jetzt in ne extra comp ausgelagert